easybuild/easyconfigs/x/XGBoost/XGBoost-1.6.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakePythonPackage'
+
+name = 'XGBoost'
+version = '1.6.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/dmlc/xgboost'
+description = """XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,
+ flexible and portable."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [{
+    'filename': 'xgboost.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/dmlc',
+        'repo_name': 'xgboost',
+        'tag': 'v1.6.0',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+
+# Can't add proper SHA256 checksum, because source tarball is created locally after recursive 'git clone'
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('CUDA', '11.3.1', '', SYSTEM),
```

Comment:
no need to make `CUDA` an explicit build dependency, since all runtime dependencies are also loaded during the build step
```suggestion
```